app/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <attr name="avatar_image_size">80dp</attr>
+</resources>
```

Comment:
I got this wrong in the last review; it should be a `dimen`, not an `attr`. My apologies.